docs/snippets/ov_extensions.py

```diff
@@ -15,6 +20,21 @@
 #! [add_extension_lib]
 core = ov.Core()
 # Load extensions library to ov.Core
-core.add_extension("openvino_template_extension.so")
+core.add_extension("libopenvino_template_extension.so")
 #! [add_extension_lib]
```

Comment:
I didn't find the documentation referring this code snippet. Do we have it?

Reply:
It already includes in our documentation